moveit_ros/planning/launch/collision_checker_compare.launch

```diff
@@ -1,10 +1,16 @@
 <launch>
+  <arg name="visualization" default="false" doc="turns on and off visualization with rviz"/>
+
   <!-- send Panda urdf to parameter server -->
   <param name="robot_description" command="$(find xacro)/xacro --inorder '$(find franka_description)/robots/panda_arm_hand.urdf.xacro'" />
 
   <include file="$(find panda_moveit_config)/launch/planning_context.launch"/>
 
   <node name="compare_collision_checking_speed" pkg="moveit_ros_planning" type="moveit_compare_collision_checking_speed_fcl_bullet" respawn="false" output="screen">
     <rosparam command="load" file="$(find panda_moveit_config)/config/kinematics.yaml"/>
+    <param name="visualization" value="$(arg visualization)" type="bool"/>
   </node>
+
+  <include if="$(arg visualization)" file="$(find panda_moveit_config)/launch/demo.launch"/>
```

Comment:
Shouldn't you provide a specific rviz config file (to be independent from changes in panda_moveit_config)?

Reply:
I see where you're coming from, but I think both the chance of a breaking change in that package and the impact in this one are so small that it's probably not worth it.